app/assets/javascripts/discourse/app/components/composer-messages.js

```diff
@@ -31,100 +31,86 @@ export default Component.extend({
     this.appEvents.on("composer:find-similar", this, this._findSimilar);
     this.appEvents.on("composer-messages:close", this, this._closeTop);
     this.appEvents.on("composer-messages:create", this, this._create);
-    scheduleOnce("afterRender", this, this.reset);
-  },
+    this.reset();
+  }
 
   willDestroyElement() {
+    this._super(...arguments);
```

Comment:
Should be `super.willDestroyElement()`, right? *and likewise for other `this._super` references

Reply:
yep yep yep!